tests/integration_tests/test_combiners.py

```diff
@@ -100,47 +118,71 @@ def encoder_comparator_outputs():
         if i == 0 or i == 3:
             dot_product_shape = [batch_shape[0], BASE_FC_SIZE]
             encoder_outputs[feature_name] = {
-                "encoder_output": tf.random.normal(dot_product_shape,
-                                                   dtype=tf.float32)
+                "encoder_output": torch.randn(dot_product_shape,
+                                              dtype=torch.float32)
             }
         else:
             encoder_outputs[feature_name] = {
-                "encoder_output": tf.random.normal(batch_shape,
-                                                   dtype=tf.float32)
+                "encoder_output": torch.randn(batch_shape,
+                                              dtype=torch.float32)
             }
 
     return encoder_outputs
 
 
 # test for simple concatenation combiner
+@pytest.mark.parametrize("number_inputs", [None, 1])
+@pytest.mark.parametrize("flatten_inputs", [True, False])
 @pytest.mark.parametrize("fc_layer",
                          [None, [{"fc_size": 64}, {"fc_size": 64}]])
```

Comment:
we can make the fc_size smaller for speed

Reply:
done.